### PR TITLE
[FIX]#1735 for ConsensusXMLFile

### PR DESCRIPTION
**Pull request by mwalzer**
see comments in #1735

### CODE REVIEW

File: src/tests/topp/FileFilter_18_output.consensusXML [hunk 1]
```diff
@@ -55,12 +53,6 @@
 	<IdentificationRun id="PI_2" date="2011-02-02T13:06:01" search_engine="Mascot" search_engine_version="2.2.04">
 		<SearchParameters db="regulatedPeptides" db_version="regulatedPeptides_appended_20100831.fasta" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="2" precursor_peak_tolerance="10" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0.5" peak_mass_tolerance_ppm="false" >
 			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Label:13C(6)15N(2) (K)" />
-			<FixedModification name="Label:13C(6)15N(4) (R)" />
```
Comment (mwalzer) [inline src/tests/topp/FileFilter_18_output.consensusXML]: The others were duplication only, but here a new ones are added! see https://github.com/OpenMS/OpenMS/blob/develop/src/tests/topp/FileFilter_18_input.consensusXML#L50

File: src/tests/topp/FileFilter_18_output.consensusXML [hunk 2]
```diff
@@ -55,12 +53,6 @@
 	<IdentificationRun id="PI_2" date="2011-02-02T13:06:01" search_engine="Mascot" search_engine_version="2.2.04">
 		<SearchParameters db="regulatedPeptides" db_version="regulatedPeptides_appended_20100831.fasta" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="2" precursor_peak_tolerance="10" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0.5" peak_mass_tolerance_ppm="false" >
 			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Label:13C(6)15N(2) (K)" />
```
Comment (timosachsenberg) [inline src/tests/topp/FileFilter_18_output.consensusXML]: so these have never been part of this SearchParameters?

**Comment by timosachsenberg** [comment]
Very nice and important fix!

File: src/tests/topp/FileFilter_18_output.consensusXML (hunk 2 above)
Comment (mwalzer) [inline src/tests/topp/FileFilter_18_output.consensusXML]: sadly, no

**Pull request merged**